YAXLib/YAXSerializer.cs

```diff
@@ -109,20 +84,11 @@ public class YAXSerializer
         /// </summary>
         private UdtWrapper _udtWrapper;
 
-        /// <summary>
-        ///     The initials used for the xml namespace
-        /// </summary>
-        private string _yaxLibNamespacePrefix = "yaxlib";
-
-        /// <summary>
-        ///     The URI address which holds the xmlns:yaxlib definition.
-        /// </summary>
-        private XNamespace _yaxLibNamespaceUri = XNamespace.Get("http://www.sinairv.com/yaxlib/");
-
         /// <summary>
         ///     Initializes a new instance of the <see cref="YAXSerializer" /> class.
         /// </summary>
         /// <param name="type">The type of the object being serialized/deserialized.</param>
+        [Obsolete("Will be removed in v4. Use YAXSerializer(Type, SerializerOptions) instead.")]
```

Comment:
I see we have an optional parameter. That's not clear from the obsolete message.

I think I prefer having 2 constructors, and without optional parameters. Optional parameters are always a bit tricky because there are inlined from the client side - although I don't think something goes wrong here. Maybe it's a bit unclear for now which constructor is called with `new YAXSerializer(type)`, this Obsolete one or the one with the optional parameter?

Reply:
Good point